src/scripts/main.js

```diff
@@ -2,7 +2,38 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach(users => {
+  const table = document.querySelector('.dashboard');
+  const newRow = document.createElement('tr');
 
-// write your code here
+  const cell1 = document.createElement('td');
```

Comment:
better to use more descriptive names
```suggestion
  const nameCell = document.createElement('td');
```